packages/wonder-blocks-search-field/src/components/search-field.stories.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import * as React from "react";
+
+import type {StoryComponentType} from "@storybook/react";
+
+import SearchField from "../index.js";
+
+export default {
+    title: "SearchField",
+};
+
+export const Default: StoryComponentType = () => {
+    const [value, setValue] = React.useState("");
+
+    const handleChange = (newValue: string) => {
+        setValue(newValue);
+    };
+
+    const handleKeyDown = (event: SyntheticKeyboardEvent<HTMLInputElement>) => {
+        if (event.key === "Enter") {
+            event.currentTarget.blur();
+        }
+    };
```

Comment:
I'm not really sure what all the use cases for `SearchField` will be. I could see there being cases where maybe pressing enter causes a dropdown and maybe in that case it might make more sense not to blur?

This is one of those things where I _think_ maybe it makes more sense to leave it for now and then add it later if it becomes relevant.